build/__dot-net-core-versions.sh

```diff
@@ -5,6 +5,6 @@ DOT_NET_CORE_21_VERSION='2.1.603'
 DOT_NET_CORE_21_SHA512='DD0EFB8AAE75D8F48EF3ABBECA38AE14D2621A47E37B2D9D74755B58F9173343305F1A62CFA9A03F17C42F58B1D1B653D271E7D1327C81FF4AF0A54C43C7DB59'
 DOT_NET_CORE_22_VERSION='2.2.202'
 DOT_NET_CORE_22_SHA512='14F5C0E6FBB874A882334E0D500E494B01D7F363028E72DB58DFFF6DB43C54670533539DCF6B8F50A97CE1E099119A8286CE84E193B361D65B1FD8C7DFFCE63D'
-DOT_NET_CORE_30_VERSION_PREVIEW_NAME='3.0.100-preview4-011223'
+DOT_NET_CORE_30_VERSION_PREVIEW_NAME='3.0.100-preview5-011568'
```

Comment:
The tests which check the dotnet core versions in the images are actually part of my another PR, so I am not adding them again here